.github/workflows/unit-test.yml

```diff
@@ -72,9 +72,15 @@ jobs:
                     - '8.1'
                     - '8.2'
                 multisite: [false, true]
+                wordpress: [''] # Latest WordPress version.
+                include:
+                    # Test with the previous WP version.
+                    - php: '7.4'
+                      wordpress: '6.0'
```

Comment:
Does that mean this string needs to be changed when a new WordPress comes out and the previous version gets outdated?

Reply:
Yeah, I would like to automate this (and actually even had written a [comment](https://github.com/WordPress/gutenberg/pull/46983#discussion_r1064706867) on it that was probably somewhat lost in rebases). I didn't want to make it a blocker for this PR though as it might get a bit involved.

(We'd likely get the `latest` version from [this WordPress.org REST API endpoint](http://api.wordpress.org/core/stable-check/1.0/), truncate the point version to get the "series", and then infer the previous series from it. E.g. `6.0.3` -> `6.0` -> `5.9`. Requires a bit of version arithmetics that I thought would be a nice fit for a separate PR. We have some similar logic e.g. [here](https://github.com/WordPress/gutenberg/blob/729efaa0bb8fa4bbff92453fefc12dfb48b3b038/.github/workflows/build-plugin-zip.yml#L38-L44).)

Reply:
It’s just a thought, but what if `wpenv` supported something like
```
`"core": "WordPress/WordPress#previous-stable",
```
or just
```
`"core": "WordPress/WordPress#previous",
```
?
This way we wouldn't have to write any custom scripts and it will work everywhere (not just in GitHub CI workflows).

Reply:
it's a good idea, but we'd probably need different syntax. That basically just converts directly to git -- e.g. github.com/WordPress/Wordpress, with `#` corresponding to a git ref (like a tag)